### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SquirrelKay**

### CODE REVIEW

File: Source/AhriAndBear/AI/BtTaskClearKey.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * 
+ */
+UCLASS()
+class AHRIANDBEAR_API UBtTaskClearKey : public UBTTaskNode
+{
+	GENERATED_BODY()
+
+	UPROPERTY(Category = "AI", EditAnywhere)
+		FName KetToClear;
```
Comment (greenstack) [inline Source/AhriAndBear/AI/BtTaskClearKey.h]: Should this be `KeyToClear`? We should probably also comment this so other devs can know its purpose.

File: Source/AhriAndBear/Characters/ShopKeeper.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	GENERATED_BODY()
+
+		UPROPERTY(Category = "Shopkeeper", VisibleDefaultsOnly)
+		class UBoxComponent* Box;
+
+	UPROPERTY(Category = "Shopkeeper", VisibleDefaultsOnly)
+		class UBoxComponent* Box1;
+
+	UPROPERTY(Category = "Shopkeeper", VisibleDefaultsOnly)
+		class USkeletalMeshComponent* SkeletalMesh;
```
Comment (greenstack) [inline Source/AhriAndBear/Characters/ShopKeeper.h]: Comments requested

File: Source/AhriAndBear/Public/Component/AABSurvivalComponent.h [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	// Called when the game starts
+	virtual void BeginPlay() override;
+
+public:	
+	// Called every frame
+	virtual void TickComponent(float DeltaTime, ELevelTick TickType, FActorComponentTickFunction* ThisTickFunction) override;
+
+	// Updates the specified stat.
+	void TickStat(float& currentValue, int maxValue, float delta, float deltaTime);
+};
```
Comment (greenstack) [inline Source/AhriAndBear/Public/Component/AABSurvivalComponent.h]: I'm currently reworking this somewhat on another branch.

File: Source/AhriAndBear/AI/BtTaskClearKey.h (hunk 1 above)
Comment (Rud156) [reply Source/AhriAndBear/AI/BtTaskClearKey.h]: Yeah. Sorry Typo I'll update it

**Review by greenstack — APPROVED** [review]

**Pull request merged**